backend/server/routes/middleware/auth.middleware.ts

```diff
@@ -0,0 +1,30 @@
+import * as express from 'express';
+
+const jwt = require('jsonwebtoken');
+
+const authMiddleware = (req: express.Request, res: express.Response, next: express.NextFunction) => {
+	console.info(`${req.method} ${req.originalUrl}`);
+	const start = new Date().getTime();
+    
+    if (req.url === '/login' || req.url === '/api/auth/signin' || req.url.includes('/api-docs')) {
```

Comment:
You can add the middleware to separate routes, as the second paramater. In this case you won't need such condition.
So in `routes.ts`: `app.post('/challenges', authMiddleware, this.challengeController.startNewChallenge)`.
And you can delete `this.application.use(authMiddleware);` in `app.ts` in this case.